src/swarm-tui/Swarm/TUI/Model/StateUpdate.hs

```diff
@@ -144,6 +145,7 @@ mkRuntimeOptions AppOpts {..} =
     { startPaused = pausedAtStart
     , pauseOnObjectiveCompletion = autoShowObjectives
     , loadTestScenarios = Set.member LoadTestingScenarios debugOptions
+    , startLogging = True
```

Comment:
Does `startLogging` ever get set to `False`?  Will the game just be always writing out a log to disk by default?

Reply:
Oh, I should add a command line option or maybe finally add a config file.